### PR TITLE
Few fixes for windows build.

### PR DESCRIPTION
**Pull request by ayermolo**
*Description*: Minor fixes to get things working again with Windows.
*Testing*: Unit Tests
*Documentation*:
[Optional Fixes #issue]

Please see a detailed explanation of how to fill out the fields in the relevant sections in PULL_REQUEST.md.

### CODE REVIEW

File: tests/unittests/OnnxImporterTest.cpp [hunk 1]
```diff
@@ -966,7 +966,7 @@ static void importPad(std::string fileName, const char *inputName,
     if (expectLoadError) {
       llvm::Error err = llvm::Error::success();
       ONNXModelLoader(NetFilename, {inputName}, {&data.getType()}, *F, &err);
-      EXPECT_TRUE(errorToBool(std::move(err)));
+      EXPECT_TRUE(glow::errorToBool(std::move(err)));
```
Comment (jackm321) [inline tests/unittests/OnnxImporterTest.cpp]: @ayermolo for my own edification, why is this needed given the `using namespace glow;` above?

Comment (ayermolo) [reply tests/unittests/OnnxImporterTest.cpp]: There is also llvm::errorToBool. VS 2017 can't figure out which one. Frankly I wasn't sure, but since using namespace glow; was used and no name space was specified assumed it was glow one.

Comment (jfix71) [reply tests/unittests/OnnxImporterTest.cpp]: Yeah I actually just hit this issue too -- the compiler complains the call is ambiguous.
